### PR TITLE
Feature/recommendations

### LINKED ISSUE #32: Write a recommendation introduction

### LINKED ISSUE #33: Extend pioneers section

### LINKED ISSUE #34: Recommendations - the process - introduction

### LINKED ISSUE #35: Recommendations - the process - hype

### LINKED ISSUE #36: Recommendations - the process - code

### LINKED ISSUE #37: Recommendations - the process - requirements

### LINKED ISSUE #38: Recommendations - the process - state

### LINKED ISSUE #39: Recommendations - integrations

### LINKED ISSUE #41: Ideal platform - goal-challanges-sub-c

### PR DESCRIPTION
**Pull request by WouterKvG**
- [x] Closes #32
- [x] Closes #33
- [x] Closes #34
- [x] Closes #35
- [x] Closes #36 
- [x] Closes #37 
- [x] Closes #38
- [x] Closes #39  
- [x] Closes #41

### CODE REVIEW

File: recommendations.md [hunk 1]
```diff
@@ -1 +1,33 @@
-#Recommendations
+# Recommendations
+No existing solutions for the problem were found. Based on the conclusions, a description is given of what an ideal solution looks like. This solution is presented in the form of a platform.
+
+## The ideal platform
+The platform converts resources, mainly in the form of time and knowledge, into an extension to the current FeedbackFruits platform. The goal of this extension should be to fulfill a particular need. A goal can be identified as the answer to finishing the phrase: "With FeedbackFruits I would like to ..". Each goal can be devided in challlenges, sub-goals that need to be met in order to meet the goal. If these challlenges are to large to be solved by one or two pioneers, then the challlenges must be devided into sub-challenges in such a way that they can be solved by one or two pioneers.
```
Comment (steffansluis) [inline recommendations.md]: can be d**i**vided

Comment (steffansluis) [inline recommendations.md]: `challlenges, sub-goals`

I would use a colon instead of a comma here

Comment (steffansluis) [inline recommendations.md]: are to**o** large

Comment (steffansluis) [inline recommendations.md]: must be d**i**vided

File: recommendations.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+## Resources
+The platform facilitates a structured process to reach a goal. On a high level, this process consists of two parts: the gathering and spending of resources. The most important resources are time and knowledge. Less important resources can include money, servers and licensed materials. Resources can be gathered by creating a hype around the goal. Once enough resources have been acquired a team of pioneers can start implementing an extenions to achieve the goal. This will result in resources being spent.
```
Comment (steffansluis) [inline recommendations.md:10]: `Less important resources`

I would say `other resources`

File: recommendations.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+
+## Pioneers
+Pioneers can be identified according to the resources they contribute. Pioneers can fall into one of three categories:
+- Core team (time and knowledge)
+- Contributors (knowledge)
+- Backers (other resources)
+
+The core team are a select group of pioneers that take the responsibility for a project. Dividing the goal into challlenges and sub-challenges is the job of the core members, thus they will provide the structure used for the software development process. Contributors can pick up one or multiple of these sub-challenges and implement the goal of those sub-challlenges. Reviewing code or designs also is a contribution to a sub-challenge. Backers are pioneers that try to improve the project without implementing or reviewing solutions for a (sub-)challlenge. This can be done with domain-specific material (like learning methods e.g.) or more generic support (like money e.g.).
```
Comment (steffansluis) [inline recommendations.md]: `thus they`

I would use `as such` here

Comment (steffansluis) [inline recommendations.md]: `structure used for`

I would remove `used`

Comment (steffansluis) [inline recommendations.md]: `domain-specific material (like learning methods e.g.) or more generic support (like money e.g.).`

There is no need to split this into these two options. I would make this a more generic list of possible contributions from backers

File: recommendations.md [hunk 4]
```diff
@@ -1 +1,33 @@
...
+
+## The process
+There are five main factors that are essential for the development of an extension: hype, requirements, code and updates. These factors correspond to different types communication between different categories of pioneers. For the development process to be clear, an iterative approach must be used. Each iterative step can be linked to a essential factor of the development. First hype must be created in order to gather necessary resources. When all the necessary resources are gathered, the requirements of the project can be created based on the goal. Code can be created based on the final requirements. The creation of code must also be done in iterations to ensure progres and code quality, where each itteration ends with an update to the contributors and the community.
```
Comment (steffansluis) [inline recommendations.md]: `five` ==> `four`

Comment (steffansluis) [inline recommendations.md]: `Each iterative step` ==> `Each iteration`

Comment (steffansluis) [inline recommendations.md]: `itteration` ==> `iteration`

**Comment by WouterKvG** [comment]
@steffansluis @bartist Anymore comments? Or can this be merged?

**Comment by bartist** [comment]
Can be merged in my opinion

File: recommendations.md [hunk 5]
```diff
@@ -1 +1,37 @@
...
+
+### Process update
+One of the most important factors in maintaining a successful community is the ability to keep everyone engaged with the common goal. To achieve this, everyone in the community should be kept up-to-date on the current state of the project, as well as future plans. In addition to keeping the community engaged, this part of the process also bridges the current iteration with the next iteration. It ends the current iteration with a summary of the results, and provides an opportunity for hype to be created around the next iteration.
+
+## Integrations
+Many of the features suggested already exist as separate tools. If such tools have an API, they could be neatly integrated into the proposed platform which prevents reinventing the wheel. For example GitHub has an extensive API with, among others, support for the creation of teams, pull-request and issues. Other integrations that can be explored are:
+- crowsourcing platforms (eg. Kickstarter, tilt.com, WePay, monster.com and gratipay);
```
Comment (steffansluis) [inline recommendations.md]: crow**d**sourcing

File: recommendations.md [hunk 6]
```diff
@@ -1 +1,37 @@
...
+
+### Process update
+One of the most important factors in maintaining a successful community is the ability to keep everyone engaged with the common goal. To achieve this, everyone in the community should be kept up-to-date on the current state of the project, as well as future plans. In addition to keeping the community engaged, this part of the process also bridges the current iteration with the next iteration. It ends the current iteration with a summary of the results, and provides an opportunity for hype to be created around the next iteration.
+
+## Integrations
+Many of the features suggested already exist as separate tools. If such tools have an API, they could be neatly integrated into the proposed platform which prevents reinventing the wheel. For example GitHub has an extensive API with, among others, support for the creation of teams, pull-request and issues. Other integrations that can be explored are:
```
Comment (steffansluis) [inline recommendations.md]: I would say `suggested features` instead of `features...suggested`

**Comment by steffansluis** [comment]
@WouterKvG I've given some small feedback. Shouldn't there be success stories part about Wikipedia and Github somewhere? The content of this PR looks good, so if it is finished feel free to merge

**Comment by WouterKvG** [comment]
@steffansluis Thanks. Weird, the success stories seem to have disappeared in some way. I'll re-add them.

**Pull request merged**